src/Tempest/Container/src/GenericContainer.php

```diff
@@ -388,13 +389,16 @@ private function autowireDependency(ParameterReflector $parameter, ?string $tag,
             return $this->autowireBuiltinDependency($parameter, $providedValue);
         }
 
+        // Support lazy initialization
+        $lazy = $parameter->getAttribute(Sometimes::class) !== null;
```

Comment:
use `->hasAttribute` instead, also maybe just inline the check instead of adding a new variable

Reply:
I didn't inline it because it's used inside a loop!

Reply:
Ah, I missed that! Carry on then :)